sklearn/tests/test_base.py

```diff
@@ -436,11 +437,11 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
         data = self.__dict__.copy()
-        data["_sklearn_version"] = sklearn.__version__
         data["_attribute_not_pickled"] = None
         return data
 
 
+@ignore_warnings(category=(DeprecationWarning, UserWarning))
```

Comment:
I think I should not be confused by `DeprecationWarning` in the title.
According to the following error message:
**UserWarning: Trying to unpickle estimator SingleInheritanceEstimator from version pre-0.18 when using version 0.19.dev0. This might lead to breaking code or invalid results. Use at your own risk.**
We just need `UserWarning`